Configuration/AlCa/python/autoCond.py

```diff
@@ -35,19 +35,16 @@
     'run2_data_relval'  :   '113X_dataRun2_relval_v4',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi' : '113X_dataRun2_PromptLike_HI_v4',
-    # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '101X_dataRun2_HLT_frozen_v11',
-    # GlobalTag for Run2 HLT: it points to the online GT
-    'run2_hlt'          :   '101X_dataRun2_HLT_frozen_v11',
-    # GlobalTag for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'      :   '112X_dataRun2_HLT_relval_v4',
-    'run2_hlt_relval_hi'   :   '112X_dataRun2_HLT_relval_HI_v3',
-    # GlobalTag for Run2 HLT for HI (not 2018 HI): it points to the online GT
-    'run2_hlt_hi'       :   '101X_dataRun2_HLTHI_frozen_v11',
+    # GlobalTag for Run3 HLT: it points to the online GT
+    'run3_hlt'          :   '112X_dataRun3_HLT_v2',
+    # GlobalTag with fixed snapshot time for Run1 HLT RelVals: customizations to run with fixed L1 Menu
+    'run1_hlt_relval'   :   '112X_dataRun2_HLT_relval_v5',
```

Comment:
Since it points anyway to the same GT as the Run2 key, what about enforcing the identity, by using the compatibility dictionary below?
Same thing is true for the `run1_data` and `run2_data` keys.

Reply:
The equality of each pair is now guaranteed by commit 988b4e0. These keys cannot be added with the compatibility keys as the `run1_hlt_relval` key is needed by `autoCondHLT` earlier in the file.

Reply:
wouldn't a re-ordering be enough here?
```diff
diff --git a/Configuration/AlCa/python/autoCond.py b/Configuration/AlCa/python/autoCond.py
index f4f08d09b5c..c94856c1d01 100644
--- a/Configuration/AlCa/python/autoCond.py
+++ b/Configuration/AlCa/python/autoCond.py
@@ -84,29 +84,11 @@ aliases = {
     'BASEGT' : 'BASE1_V1|BASE2_V1'
 }
 
-### Run 1 data GTs ###
+### OLD KEYS ### kept for backward compatibility
     # GlobalTag with fixed snapshot time for Run1 HLT RelVals: customizations to run with fixed L1 Menu
 autoCond['run1_hlt_relval']  = autoCond['run2_hlt_relval']
     # GlobalTag for Run1 data reprocessing
 autoCond['run1_data']        = autoCond['run2_data']
-
-# dedicated GlobalTags for HLT
-from Configuration.HLT.autoCondHLT import autoCondHLT
-autoCond = autoCondHLT(autoCond)
-
-# dedicated GlobalTags for phase-2 (specializing conditions for each geometry)
-from Configuration.AlCa.autoCondPhase2 import autoCondPhase2
-autoCond = autoCondPhase2(autoCond)
-
-# special cases modifier for autoCond GTs
-from Configuration.AlCa.autoCondModifiers import autoCond0T
-autoCond = autoCond0T(autoCond)
-
-# special GT for 2015 HLT HI run
-from Configuration.AlCa.autoCondModifiers import autoCondHLTHI
-autoCond = autoCondHLTHI(autoCond)
-
-### OLD KEYS ### kept for backward compatibility
     # GlobalTag for MC production with perfectly aligned and calibrated detector
 autoCond['mc']               = ( autoCond['run1_design'] )
     # GlobalTag for MC production with realistic alignment and calibrations
@@ -125,3 +107,19 @@ autoCond['upgradePLS150ns']  = ( autoCond['run2_mc_50ns'] )
 autoCond['upgrade2017']      = ( autoCond['phase1_2017_design'] )
 autoCond['upgrade2021']      = ( autoCond['phase1_2021_design'] )
 autoCond['upgradePLS3']      = ( autoCond['phase2_realistic'] )
+
+# dedicated GlobalTags for HLT
+from Configuration.HLT.autoCondHLT import autoCondHLT
+autoCond = autoCondHLT(autoCond)
+
+# dedicated GlobalTags for phase-2 (specializing conditions for each geometry)
+from Configuration.AlCa.autoCondPhase2 import autoCondPhase2
+autoCond = autoCondPhase2(autoCond)
+
+# special cases modifier for autoCond GTs
+from Configuration.AlCa.autoCondModifiers import autoCond0T
+autoCond = autoCond0T(autoCond)
+
+# special GT for 2015 HLT HI run
+from Configuration.AlCa.autoCondModifiers import autoCondHLTHI
+autoCond = autoCondHLTHI(autoCond)

```

Reply:
I prefer the current ordering as the keys `run1_hlt_relval` and `run1_data` are standard `autoCond` keys, even though they each are fixed to have the same value as another key. I would rather keep the deprecated keys separate at the end of the file.